gsa/src/gmp/models/__tests__/nvt.js

```diff
@@ -154,6 +154,43 @@ describe('nvt Model tests', () => {
     expect(nvt3.severity).toEqual(9.5);
   });
 
+  test('should parse severity and origin from severities', () => {
+    const nvt1 = Nvt.fromElement({
+      severities: {
+        severity: {
+          score: 94,
+          origin: 'Vendor',
+        },
+      },
+    });
+    const nvt2 = Nvt.fromElement({
+      severities: {
+        severity: {
+          score: 74,
+          origin: 'Greenbone',
+        },
+      },
+    });
+    const nvt3 = Nvt.fromElement({
+      severities: {
+        severity: {
+          score: 10,
+          origin: 'Third Party',
```

Comment:
Can origin be undefined or empty? We should have one nvt in the test where we don't have an origin to make sure we can handle that correctly.